frontend/tsconfig.json

```
@@ -41,7 +41,8 @@
       "@pages": ["pages/index.ts"],
       "@pages/*": ["pages/*"]
     },
-    "typeRoots": ["src/custom-types"]
+    "typeRoots": ["src/custom-types"],
+    "types": ["cypress", "@testing-library/cypress"]
   },
-  "include": ["src"]
+  "include": ["src", "node_modules/cypress", "cypress/**/*.cy.ts", "cypress/**/*.cy.tsx"]
```

```suggestion
  "include": ["src", "node_modules/cypress", "cypress/**/*.cy.ts", "cypress/**/*.cy.tsx"]
```

`node_modules/cypress`도 include 해야 하나요?